components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/service/APIGatewayAdmin.java

```diff
@@ -776,10 +776,6 @@ public boolean deployAPI(GatewayAPIDTO gatewayAPIDTO) throws AxisFault {
             log.debug("Start to deploy Default API Definition" + gatewayAPIDTO.getName() + ":" + gatewayAPIDTO.getVersion());
         }
 
-        // Add Default API
-        if (StringUtils.isNotEmpty(gatewayAPIDTO.getDefaultAPIDefinition())) {
-            restapiAdminServiceProxy.addApi(gatewayAPIDTO.getDefaultAPIDefinition());
```

Comment:
is this effective only for rest ?, we still need default api for websocket

Reply:
we might have to fix WebSocket in the same way

Reply:
done